src/hooks/useAxios.js

```diff
@@ -0,0 +1,24 @@
+import axios from "axios";

```

Comment:
This hook is not used anywhere in the project. You can remove it, if you are thinking to use it in the future you can place this file in .gitignore. You should use this it will save lots of code repetition